pkg/asset/tls.go

```diff
@@ -22,6 +22,34 @@ func newTLSAssets(caCert *x509.Certificate, caPrivKey *rsa.PrivateKey, altNames
 		return assets, err
 	}
 
+	aggregatorCAPrivKey, err := tlsutil.NewPrivateKey()
+	if err != nil {
+		return assets, err
+	}
+
+	cfg := tlsutil.CertConfig{
+		CommonName: "front-proxy",
+	}
+
+	aggregatorCACert, err := tlsutil.NewSelfSignedCACertificate(cfg, aggregatorCAPrivKey)
+	if err != nil {
+		return assets, err
+	}
+
+	frontProxyPrivKey, err := tlsutil.NewPrivateKey()
+	if err != nil {
+		return assets, err
+	}
+
+	cfg = tlsutil.CertConfig{
+		CommonName: "front-proxy-client",
```

Comment:
nit : is this a standard naming convention ? `front-proxy-*`

Reply:
It is. At least in kubeadm.